src/flows/pipes/Notification/Expression.tsx

```diff
@@ -0,0 +1,50 @@
+// Libraries
+import React, {FC, createRef} from 'react'
+
+// Component
+import {Button, ComponentSize, ComponentColor} from '@influxdata/clockface'
+
+interface Props {
+  expression: string
+  onClickFunction: (expression: string) => void
+  testID: string
+}
+
+const defaultProps = {
+  testID: 'alert-expression',
+}
+
+const ToolbarExpression: FC<Props> = ({
+  expression,
+  onClickFunction,
+  testID,
+}) => {
+  const expressionRef = createRef<HTMLDListElement>()
+  const handleClickFunction = () => {
+    onClickFunction(expression)
+  }
+
+  return (
+    <>
+      <dd
+        ref={expressionRef}
```

Comment:
what is this for?

Reply:
lol no idea. copy pasta. Seems to work fine without it so I'll throw it away